CONTRIBUTING.md

```diff
@@ -123,7 +123,15 @@ and additionally the following rules:
   reviewers fall in love with you :) https://mtlynch.io/code-review-love/
   Keep in mind that the text in the github pull request description is only
   visible on github, not in the git log which can be considered permanent
-  information storage.
+  information storage.  
+  Commits will be checked automatically by [this workflow defined in os-autoinst/os-autoinst-common](https://github.com/os-autoinst/os-autoinst-common/blob/master/.github/workflows/base-commit-message-checker.yml) to enforce the following rules:
```

Comment:
please wrap lines consistently at 80 characters.

Also I would state that the following rules are just examples, not necessarily all the rules because we might extend the git commit message check and not update all the rules stated here.

Reply:
Addressed in d08199e